### DISCUSSION TITLE
16. HTTP와 HTTPS의 차이점에 대해서 설명해 주세요.

### DISCUSSION THREAD

**Discussion by cutehumanS2** [jewelry-box/cs-interview#3] [General]
HTTP와 HTTPS의 차이점에 대해서 설명해 주세요.

**Comment by jongmee** [comment]
- HTTPS는 HTTP에서 `SSL/TLS`로 데이터를 `암호화`해서 보안을 강화한 프로토콜이다. 데이터를 암호화해서 도청이나 중간자 공격을 방지한다. 또한 전송된 데이터가 도중에 변조되지 않았음을 보장하고 웹사이트가 신뢰할 수 있는 기관에 의해 인증되었음을 보장한다.

- HTTPS는 연결할 때 TCP 3 way handshake 이후에 SSL handshake를 진행한다.
